content/working_with/console/widgets/userManagement.md

```diff
@@ -23,9 +36,56 @@ The hamburger menu on the right of every tenant allows performing the following
+Please notice that if you choose to authenticate the users in front of an external user management system, you will not be able to create or delete the users in cloudify, nor to assigned them to Cloudify user groups,  to prevent conflicts between the two systems which might cause security problems. 
+
+#### Adding Users
+
+1. Click **Add** button in the widget.
+2. Specify a name for the user.   
+   The name must contain at least five alphanumeric characters and begin with a letter. It can also include the following characters `-`, `_`, or `.`.
+3. Specify a password for the user, then confirm it.   
+   The password must contain at least five alphanumeric characters and begin with a letter. It can also include the following characters `-`, `_`, or `.`.
+4. Check **Admin** checkbox to assign the user administrator privileges. In order to have access to specific tenants in the system, the user will also need to be explicitly assigned with those tenants, and have a tenant-specific role for each of them. For more information about the roles and the differences between them, see Roles-Management section.
+5. Select tenant(s) from **Tenants** list and assign a tenant-specific role(s) for the user.
+6. Click **Add**.
+
+
+#### Assigning Users to Tenants
+
+Starting Cloudify 4.2, users must be assigned to tenants with a specific role for each tenant. The roles are sets of permissions defining what actions the users can perform in the context of the tenant. Users can have different roles in different tenants, and as long as at least one of those roles allows them to perform an action in the tenant, they will be able able to perform it. The available tenant-roles are: viewer, user, operations and manager. For more information regarding each role and the differences between them, see [Roles Management section]({{< relref "working_with/manager/user-management.md#roles-management-with-ldap" >}}).
+
+1. Click the List icon ![List icon]( /images/ui/icons/list-icon.png ) on the far right of the user entry in the table that you want to add to a tenant.
+2. Click **Edit user's tenants**.
+3. Select one or more tenants from the dropdown list.
+4. Select role for each tenant.
+5. Click **Save**.
+
+The user is added to the specified tenants.   
+Unless the users have a deactivated status, they can perform actions on the tenant according to the specified role.
+
+![User options]( /images/ui/widgets/users_tenant_role.png )
+
+
+#### Removing a User from a Group or Tenant
+
+You can remove a user from a group or a tenant, without deleting them from the system.
+
+1. Click the List icon ![List icon]( /images/ui/icons/list-icon.png ) of the user that you want to remove.
+2. Select **Edit user's groups** or **Edit user's Tenants**. 
+3. Remove the desired groups/tenants from the users's list.
+4. Click **Save**.
+
+The user is removed. If a user is a member of one or more user groups that are still assigned to a tenant, that user remains active on the tenant.
```

Comment:
"The user is removed." - this is not the case.

Reply:
Fixed.